### ISSUE TITLE
`--filterLowQual` does not work

### ISSUE THREAD

**Issue by bounlu** [bioinfo-pf-curie/TMB#21]
It works fine without the filter `--filterLowQual`. When I add this filter, no variant is selected and TMB is always 0. It seems there is a bug with the filtering.

**Comment by nservant** [comment]
Hi,
Thanks for the comment.
The `--filterLowQual` filter only keep `PASS` variants. Do you have a `PASS` flag in your VCF ?

**Comment by bounlu** [comment]
Yes I do and that's also my expectation but it somehow doesn't work.

**Comment by bounlu** [comment]
Here is the bug:
https://github.com/bioinfo-pf-curie/TMB/blob/97ee724ac5860822ec59177e43456634067d652a/bin/pyTMB.py#L409-L413

**Comment by kiranchari** [comment]
@nservant when using why args.filterLowQual , why are variants filtered when QUAL is not None?

**Comment by nservant** [comment]
because if QUAL is None it means that the caller was not able to assess the variant quality ? does it make sense ?

**Comment by kiranchari** [comment]
This line is doing the opposite, it filters variants if QUAL is defined (i.e. not None).

https://github.com/bioinfo-pf-curie/TMB/blob/97ee724ac5860822ec59177e43456634067d652a/bin/pyTMB.py#L410

However, not all callers report a QUAL score e.g. Mutect2 does not report a QUAL score (it just reports `.` which would be converted to None).

**Comment by nservant** [comment]
fixed in 1.5.0
